src/qt/forms/overviewpage.ui

```diff
@@ -549,12 +549,57 @@
          </property>
         </spacer>
        </item>
+       <item>
+        <layout class="QVBoxLayout" name="verticalLayout_5">
+         <property name="topMargin">
+          <number>0</number>
+         </property>
+         <item>
+          <widget class="QLabel" name="label_tip_header">
+           <property name="font">
+            <font>
+             <weight>75</weight>
+             <bold>true</bold>
+            </font>
+           </property>
+           <property name="text">
+            <string>Helpful tip of the day:</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QLabel" name="label_tip">
+           <property name="text">
+            <string>Tip</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <spacer name="verticalSpacer_3">
+           <property name="orientation">
+            <enum>Qt::Vertical</enum>
+           </property>
+           <property name="sizeType">
+            <enum>QSizePolicy::Maximum</enum>
+           </property>
+           <property name="sizeHint" stdset="0">
+            <size>
+             <width>20</width>
+             <height>40</height>
```

Comment:
Don't know if it's possible in Qt Forms, but we might want to use variables to have the interface responsive.

Reply:
The other vertical spacers in the form use the same size width and height, so I used those numbers for the new vertical spacer.

Reply:
Here is the image of the full-screen version:
![dogetipfull](https://user-images.githubusercontent.com/36016500/121940044-af12fb80-cd1b-11eb-9bf6-5058f23a62bb.png)

Reply:
Right, all values are currently hard coded. It looks fine to me for this PR. It might be something to consider for future work or new issue.